packages/serverpod_shared/lib/src/config.dart

```diff
@@ -126,6 +130,15 @@ class ServerpodConfig {
           )
         : null;
 
+    var sessionLogsConfig = _sessionLogsConfigMap(configMap, environment);
+    var losessionLogs = sessionLogsConfig != null
```

Comment:
Suggestion: Rename the first variable to `sessionLogsConfigJson` and the second one to `sessionLogsConfig`